roles/mastodon/templates/nginx/mastodon.conf.j2

```diff
@@ -45,7 +45,7 @@ server {
   gzip_http_version 1.1;
   gzip_types text/plain text/css application/json application/javascript text/xml application/xml application/xml+rss text/javascript;
 
-{% if disable_hsts == "true" %}
+{% if disable_hsts != "true" %}
```

Comment:
```suggestion
{% if disable_hsts | bool %}
```
[https://docs.ansible.com/ansible/latest/user_guide/playbooks_filters.html#id8](https://docs.ansible.com/ansible/latest/user_guide/playbooks_filters.html#id8)

I think that's a more elegant solution